core/src/main/java/ar/com/wolox/wolmo/core/util/Logger.kt

```diff
@@ -0,0 +1,45 @@
+package ar.com.wolox.wolmo.core.util
+
+import android.util.Log
+
+import javax.inject.Inject
+
+/**
+ * Wrapper of [Log] to simplify logs in the same class by reusing the tag and
+ * to simplify unit tests.
+ */
+class Logger @Inject constructor() {
+
+    /** [tag] used to identify the source of a log message. */
+    var tag: String? = null
+
+    /** Send a [Log.VERBOSE] log a [message] and an [exception] with a [tag] to identify the source of it. */
+    fun v(tag: String?, message: String, exception: Throwable? = null) = Log.v(tag, message, exception)
+
+    /** Send a [Log.VERBOSE] log a [message] and an [exception] using the predefined [tag]. */
+    fun v(message: String, exception: Throwable? = null) = v(tag, message, exception)
+
+    /** Send a [Log.DEBUG] log a [message] and an [exception] with a [tag] to identify the source of it. */
+    fun d(tag: String?, message: String, exception: Throwable? = null) = Log.d(tag, message, exception)
+
+    /** Send a [Log.DEBUG] log a [message] and an [exception] using the predefined [tag]. */
+    fun d(message: String, exception: Throwable? = null) = d(tag, message, exception)
+
+    /** Send a [Log.INFO] log a [message] and an [exception] with a [tag] to identify the source of it. */
+    fun i(tag: String?, message: String, exception: Throwable? = null) = Log.i(tag, message, exception)
+
+    /** Send a [Log.INFO] log a [message] and an [exception] using the predefined [tag]. */
+    fun i(message: String, exception: Throwable? = null) = i(tag, message, exception)
+
+    /** Send a [Log.WARN] log a [message] and an [exception] with a [tag] to identify the source of it. */
+    fun w(tag: String?, message: String, exception: Throwable? = null) = Log.w(tag, message, exception)
+
+    /** Send a [Log.WARN] log a [message] and an [exception] using the predefined [tag]. */
+    fun w(message: String, exception: Throwable? = null) = w(tag, message, exception)
+
+    /** Send a [Log.ERROR] log a [message] and an [exception] with a [tag] to identify the source of it. */
+    fun e(tag: String?, message: String, exception: Throwable? = null) = Log.e(tag, message, exception)
+
+    /** Send a [Log.ERROR] log a [message] and an [exception] using the predefined [tag]. */
```

Comment:
Why use multiline comments (/* */) for just one line instead of // ?
And even if so, maybe it would look better if you removed the second `*` after `/*`?

Reply:
Just asking, because I noticed it looks like that on the rest of the PRs as well.

Reply:
It's like a convention of how to make documentation: https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments 
In fact, they don't do oneline commets, but I think is a good idea, what do you think?